math/lean/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        leanprover-community lean 3.32.1 v
+revision            0
+
+categories          math
+license             Apache-2
+maintainers         {@kakuhen} openmaintainer
+platforms           darwin
+supported_archs     x86_64 arm64
+description         The Lean theorem prover.
+long_description    A fork of Lean 3 maintained and updated by the Lean \
+    community. Lean is a functional programming language that makes it easy to \
+    write correct and maintainable code. You can also use Lean as an \
+    interactive theorem prover.
+
+depends_build       port:cmake port:gmp
```

Comment:
A port specified in `depends_lib` does not have to be specified again in `depends_build`.

If a library is needed **only** at runtime, then it would get specified in `depends_run`.

Something in `depends_lib` is understood to be needed both at build *and* run -time.

Reply:
Ah, just saw @kencu already commented on this 😅

Reply:
Both comments gave me good information, for this port and any future ports I may submit. Thanks to you two for the feedback. If there is anything else, please let me know.